akka-cluster/src/test/scala/akka/cluster/CrossDcHeartbeatSenderSpec.scala

```diff
@@ -36,18 +41,25 @@ class CrossDcHeartbeatSenderSpec extends AkkaSpec("""
   """) with ImplicitSender {
   "CrossDcHeartBeatSender" should {
     "increment heart beat sequence nr" in {
-      val probe = TestProbe()
+
+      val heartbeatProbe = TestProbe()
       Cluster(system).join(Cluster(system).selfMember.address)
       awaitAssert(Cluster(system).selfMember.status == MemberStatus.Up)
-      val underTest = system.actorOf(Props(new TestCrossDcHeartbeatSender(probe)))
+      val underTest = system.actorOf(Props(new TestCrossDcHeartbeatSender(heartbeatProbe)))
+
       underTest ! CurrentClusterState(
         members = SortedSet(
           Cluster(system).selfMember,
           Member(UniqueAddress(Address("akka", system.name), 2L), Set("dc-dc2"), Version.Zero)
             .copy(status = MemberStatus.Up)))
 
-      probe.expectMsgType[Heartbeat].sequenceNr shouldEqual 1
-      probe.expectMsgType[Heartbeat].sequenceNr shouldEqual 2
+      awaitAssert {
+        underTest ! ReportStatus()
+        expectMsgType[CrossDcHeartbeatSender.MonitoringActive]
+      }
```

Comment:
that's the fix, the rest is just formatting and val renaming. 

We need to first confirm that `CrossDcHeartbeatSender` is active before we can assert on heartbeat.

Reply:
note, the block is marked as "Outdated" because I move the sending of ReportStatus to outside the awaitAssert block